apps/examples/timer/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
+# For a description of the syntax of this configuration file,
+# see misc/tools/kconfig-language.txt.
```

Comment:
There is no misc folder. Use below.
"# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt"